json-resume/src/App.js

```diff
@@ -15,14 +18,12 @@ function App() {
     <Router>
     <div className="App">
     <header className="App-header">
-    
-        
-       
-    
+
     <Switch>
       <Route exact path="/">
-        <DataEntry />
+        <DataEntry resume={resume} />
```

Comment:
are you rendering twice here so you have one blank and one filled our for editing purposes?